packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -1785,7 +1785,8 @@ export class DataViewExpander {
             image: "",
             type: "RelationshipDVO",
             status: relationship.status,
-            peerDeletionInfo: relationship.peerDeletionInfo,
+            peerDeletionStatus: relationship.peerDeletionInfo?.deletionStatus,
+            peerDeletionDate: relationship.peerDeletionInfo?.deletionDate,
```

Comment:
I had in mind to change the name of the property within peerDeletionInfo itself, too. I suppose like this it works as well. I'm not completely certain what I like more. @Magnus-Kuhn Do you have a strong opinion on this?

Reply:
I liked it the way it was done in the attributes (like this), and generally it's no effort to move to the long version should the need arise, so yagni - not a particularly strong opinion though